### PR TITLE
Add a README with installation instructions

### PR DESCRIPTION
**Pull request by jwoudenberg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Brangelina.vim
+
+## Installation instructions
+- [Install Neovim, along with the Python2, Python3, and Ruby modules](https://github.com/neovim/neovim/wiki/Installing-Neovim)
+- Run `:Checkhealth` to ensure everything is working correctly
```
Comment (avh4) [inline README.md]: I think this requires a capital `H`

**Comment by avh4** [comment]
Looks like a good start.  (Lots of steps, though! :) )

File: README.md (hunk 1 above)
Comment (stoeffel) [reply README.md]: yes

**Comment by stoeffel** [comment]
LGTM

**Comment by avh4** [comment]
FYI, Brooke followed it this afternoon, and it seemed to all be correct.

**Pull request merged**